setup.py

```diff
@@ -51,7 +51,8 @@
         'ftw.profilehook',
         'Pillow',
         'ftw.colorbox',
-        'plone.api'
+        'plone.api',
+        'plone.batching>=1.0.4'
```

Comment:
@jone Tested with earlier version (plone.batching 1.0). It works without issue.
I removed the dependency.